t/Analizo/Batch/Output/CSV.t

```diff
@@ -109,8 +109,6 @@ sub must_return_short_names_of_metrics : Tests {
 	ok($short_names[0] eq "acc", "must list acc metric name");
 	ok($short_names[1] eq "accm", "must list accm metric name");
 	ok($short_names[2] eq "amloc", "must list amloc metric name");
-	ok($short_names[3] eq "an", "must list an metric name");
-	ok($short_names[4] eq "anpm", "must list anpm metric name");
```

Comment:
It was a mistake, we'll fix it and submit again. It's short names are a little confusing. Do you have any report else?